app/controllers/admin/events_controller.rb

```diff
@@ -1,65 +1,67 @@
-class Admin::EventsController < ApplicationController
-  include EventsHelper
+module Admin
+  class EventsController < ApplicationController
+    include EventsHelper
 
-  before_action :ensure_admin
-  before_action :set_event, only: [:show, :edit, :update, :destroy]
+    before_action :ensure_admin
+    before_action :set_event, only: [:show, :edit, :update, :destroy]
 
-  # GET /events
-  def index
-    @events = Event.all.order(year: :desc)
-  end
+    # GET /events
+    def index
+      get_all_event
+    end
 
-  # GET /events/1
-  def show
-  end
+    # GET /events/1
```

Comment:
can you please remove all of misleading comments like this :) or perhaps make it correct, because we don't query by id anymore right hahaha

Reply:
Ok, I just removed the misleading comments :)